src/app/home/events/events.component.html

```diff
@@ -0,0 +1,27 @@
+<div class="event-container section center-align">
+  <div class="row event-container">
```

Comment:
I still don't get why you have to repeat style "event-container" twice. Also, why you are not using "container" class for this component template. All components with padding left and right use container class in order to have the same padding across all the app. In the morning I will checkout your branch to understand this better.